Sources/PointFreeTestSupport/TestCase.swift

```diff
@@ -16,13 +17,24 @@ open class TestCase: XCTestCase {
     Backtrace.install()
   }
 
+  open override func invokeTest() {
+    withDependencyValues {
+      $0.database = .mock
+      $0.date.now = .mock
+      $0.envVars = $0.envVars.assigningValuesFrom(ProcessInfo.processInfo.environment)
+      $0.gitHub = .mock
+      $0.mailgun = .mock
+      $0.stripe = .mock
+      $0.uuid = .incrementing
+    } operation: {
+      super.invokeTest()
+    }
```

Comment:
This is essentially what we are doing on main, but I think eventually we would like to remove all these and be more explicit in tests that are actually using one of these dependencies.